Commands/MakeCommand.php

```diff
@@ -22,13 +22,40 @@ class MakeCommand extends BaseCommand
      * @var string
      */
     protected $name = 'make:command';
+    
 
     /**
      * the Command's short description
      *
      * @var string
      */
-    protected $description = 'Creates a skeleton command file.';
+    protected $description = 'Creates a skeleton command file.';    
+
+     /**
+     * the Command's usage
+     *
+     * @var string
+     */
+    protected $usage = 'make:command [command_name] [Options]';
+
+    /**
+     * the Command's Arguments
+     *
+     * @var array
+     */
+    protected $arguments = array(
```

Comment:
While there's no real style guide for this project, I would prefer to use short array syntax and, in general, keep it close to CI4's styles.